guardian/commercial-templates
# Upgrade node to v16 in legacy

## What does this change?

- upgrade Node to v16.3.1

## How to test

```
$ cd legacy
$ npm install
$ npm run build
$ npm run preview
```

Observe that no errors appear 🪄

## How can we measure success?

This change synchronises Node versions in `legacy` and `svelte` projects. It is less developer overhead having to worry about different Node versions in this repository.

Also I was having trouble downloading Node v14 using `asdf`. Installation was either failing or hanging. I had no such problem with v16.
